src/valkey/js_valkey_functions.zig

```diff
@@ -582,7 +582,7 @@ pub const bitcount = compile.@"(key: RedisKey)"("bitcount", "BITCOUNT", "key").c
 pub const dump = compile.@"(key: RedisKey)"("dump", "DUMP", "key").call;
 pub const expiretime = compile.@"(key: RedisKey)"("expiretime", "EXPIRETIME", "key").call;
 pub const getdel = compile.@"(key: RedisKey)"("getdel", "GETDEL", "key").call;
-pub const getex = compile.@"(key: RedisKey)"("getex", "GETEX", "key").call;
+pub const getex = compile.@"(...strings: string[])"("getex", "GETEX").call;
```

Comment:
i wonder the perf difference of this, also does it handle no/less params passed well?

Reply:
it'll be the same